### PR TITLE
[17.0][MIG]sale_fixed_discount: Migrated module to v17.0

### PR DESCRIPTION
**Pull request by fd-oerp**

### CODE REVIEW

**Comment by fd-oerp** [comment]
Hi @OCA-git-bot, can I have any update regarding this PR?

**Comment by fd-oerp** [comment]
Hi @JordiBForgeFlow , could you please review this PR ?

File: sale_fixed_discount/README.rst [hunk 1]
```diff
@@ -42,52 +42,61 @@ Installation
...
 
 Contributors
-~~~~~~~~~~~~
+------------
 
-* Lois Rilo <lois.rilo@forgeflow.com> (www.forgeflow.com)
-* Jordi Ballester <jordi.ballester@forgeflow.com> (www.forgeflow.com)
-* Pieter Paulussen <pieterpaulussen@code-source.be> (www.code-source.be)
+-  Lois Rilo <lois.rilo@forgeflow.com>
+   (`www.forgeflow.com <http://www.forgeflow.com>`__)
```
Comment (JordiBForgeFlow) [inline sale_fixed_discount/README.rst:88]: change to https

File: sale_fixed_discount/README.rst [hunk 2]
```diff
@@ -42,52 +42,61 @@ Installation
...
+-  Jordi Ballester <jordi.ballester@forgeflow.com>
+   (`www.forgeflow.com <http://www.forgeflow.com>`__)
```
Comment (JordiBForgeFlow) [inline sale_fixed_discount/README.rst:90]: change to https

File: sale_fixed_discount/README.rst [hunk 3]
```diff
@@ -42,52 +42,61 @@ Installation
...
+-  Pieter Paulussen <pieterpaulussen@code-source.be>
+   (`www.code-source.be <http://www.code-source.be>`__)
```
Comment (JordiBForgeFlow) [inline sale_fixed_discount/README.rst:92]: same

File: sale_fixed_discount/readme/INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+This module depends on module 'account_invoice_fixed_discount',
+available in
+<https://github.com/OCA/account-invoicing/tree/16.0/account_invoice_fixed_discount>
```
Comment (JordiBForgeFlow) [inline sale_fixed_discount/readme/INSTALL.md]: Change to 17.0

**Review by JordiBForgeFlow — CHANGES_REQUESTED** [review]

File: sale_fixed_discount/README.rst (hunk 1 above)
Comment (fd-oerp) [reply sale_fixed_discount/README.rst:88]: Hi @JordiBForgeFlow ,
if I chnage it to 'https' precommit fails and make it to 'http' by it self
I'm running precommit through : pre-commit run -a
would that be fine to commit it with 's' ?

Comment (JordiBForgeFlow) [reply sale_fixed_discount/README.rst:88]: Does pre-commit give you any error message? which one is it?

Comment (fd-oerp) [reply sale_fixed_discount/README.rst:88]: Hi @JordiBForgeFlow 
Generate addons README files from fragments..............................Failed
- hook id: oca-gen-addon-readme
- files were modified by this hook

is this, and it update 'https' to 'http' by it self, still should I proceed with https ?

Comment (JordiBForgeFlow) [reply sale_fixed_discount/README.rst:88]: Leave it for now and attend to my other comments, please.

**Comment by JordiBForgeFlow** [comment]
Please merge your last two commits together using git rebase -i origin/17.0

File: sale_fixed_discount/readme/INSTALL.md (hunk 4 above)
Comment (fd-oerp) [reply sale_fixed_discount/readme/INSTALL.md]: Hi @JordiBForgeFlow , This changes has been updated to this commit : [2287d4d069bbc3b240bd48189efcbfde57beb4b9](https://github.com/OCA/sale-workflow/pull/2952/commits/2287d4d069bbc3b240bd48189efcbfde57beb4b9)
please review

**Comment by fd-oerp** [comment]
> Please merge your last two commits together using git rebase -i origin/17.0

@JordiBForgeFlow commits are merged now

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by bpcreate** [comment]
probably should update odoo apps page to say this works for version 17. :)

**Pull request merged**